Week7/Inventory.java

```diff
@@ -0,0 +1,36 @@
+import java.util.HashMap;
+
+
+public class Inventory
+{
+	HashMap<Integer, Item> items;
+	
+	public Inventory(){
+		Item item1 = new Item(1.25, "Peanuts", 10, 1);
+		Item item2 = new Item(1.00, "Granola Bar", 7, 2);
+		Item item3 = new Item(1.50, "Cookies", 1, 3);
+		
+		items = new HashMap<Integer, Item>();
+		items.put(1, item1);
+		items.put(2, item2);
+		items.put(3, item3);
+	}
+	
+	public void deleteItem(int itemId) throws SoldOutItemException{
+		if (items.get(itemId).getItemCount() == 0){
```

Comment:
What if this "items.get(itemId).getItemCount()" statement evaluates to a negative number ? then we will subtract from that number instead of throwing exception. 

It' god to be pessimist in these situations, and assume that we might end up getting negative values, or 0 or positive values, and handle all these cases. Ideally, the code would be:

if(<item count > <= 0)
     throw exception
else
    decrease the count here.